### ISSUE TITLE
GVim show nothing in the window

### ISSUE THREAD

**Issue by bobcao3** [AOSC-Dev/aosc-os-abbs#512]
When I edit a file with GVim, the poped out window is white without anything. Here is a bug tracked:

`# gvim Gy.java`
`(gvim:5392): Gtk-CRITICAL **: gtk_widget_set_size_request: assertion 'width >= -1' failed`
`*** BUG ***`
`In pixman_region32_init_rect: Invalid rectangle passed`
`Set a breakpoint on '_pixman_log_error' to debug`

**Comment by MingcongBai** [comment]
Found a reference here https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=842819. Probably dropping back to Gtk2 - I am looking into it.

**Comment by MingcongBai** [comment]
Fixed with 8.0.0095:

```
patch 8.0.0095
Problem:    Problems with GTK 3.22.2 fixed in 3.22.4.
Solution:   Adjust the #ifdefs. (Kazunobu Kuriyama)
```

I am bumping to 0101, in conjunction with https://github.com/AOSC-Dev/aosc-os-abbs/issues/508 (a security update).
